src/shared/codelens/localLambdaRunner.ts

```diff
@@ -510,14 +549,19 @@ export const invokeLambdaFunction = async (params: {
         invoker: params.samLocalInvokeCommand,
     })
 
-    const startInvokeTime = new Date()
-    await command.execute()
+    const timelimit = params.configuration.readSetting<number>(
+        'samcli.debug.attach.timeout.millis',
+        SAM_LOCAL_PORT_CHECK_RETRY_TIMEOUT_MILLIS_DEFAULT
+    )
+    const timer = new InvokeTimer(timelimit ? timelimit : SAM_LOCAL_PORT_CHECK_RETRY_TIMEOUT_MILLIS_DEFAULT)
```

Comment:
See above, we shouldn't need to restate the default.

Reply:
This was due to the linter not liking the `readSetting` implementation. Changing it on that end.

Reply:
✔️